guides/plugins/apps/hosting-guide/platform-sh-deployment.md

```diff
@@ -0,0 +1,115 @@
+# Platform.sh Deployment
+
+## Overview
+
+[Platform.sh](https://platform.sh) is a powerful hosting provider for your infrastructure that's quite easy to use.
+
+Keep in mind though, that this is **not** the only way to go for Apps! You can of course use different services, provider or host everything on a dedicated machine.
+In this guide we just tell you what we would do to start hosting an App von Platform.sh.
+
+Read more about why this kind of hosting could be useful [here](README.md) or on there [official documentation](https://docs.platform.sh/).
+
+## Getting started
+To deploy your app on [Platform.sh](https://platform.sh), just follow the instructions:
+
+* [Source Integrations](https://docs.platform.sh/integrations/source.html)
+* [Private Git repository](https://docs.platform.sh/development/private-repository.html)
+* [Using the Platform.sh CLI](https://docs.platform.sh/development/cli.html)
+
+## Most important steps
+
+1. Configure your [Source Integrations](https://docs.platform.sh/integrations/source.html) (Optional, but highly recommended!)
+2. Install the [Platform.sh CLI](https://docs.platform.sh/development/cli.html)
+3. [Authenticate](https://docs.platform.sh/development/cli.html#authentication) using your Platform.sh account
+4. Create required config files. Also, if you create a new project, Platform.sh shows you a checklist where you can generate the code for these files
+    * [routes.yaml](https://docs.platform.sh/configuration/routes.html)
+    * [services.yaml](https://docs.platform.sh/configuration/services.html)
+    * [.platform.app.yaml](https://docs.platform.sh/configuration/app.html)
+5. Push your changes to your Git Repo
+6. After it's deployed, migrate the database by connecting via [SSH to your project](#ssh-into-your-project) and running the command `vendor/bin/doctrine-migrations migrations:migrate`
+7. That's it!
+
+Your project should now be running at [https://console.platform.sh](https://console.platform.sh), and you can start developing your own app!
+
+## Good to know
+
+### Automatic TLS certificates based on Branch / Pull Request
+[Platform.sh](https://platform.sh) automatically creates a URL and TLS certificate using [Let's Encrypt](https://letsencrypt.org/) based on your [routes.yaml](https://docs.platform.sh/configuration/routes.html) file for every active environment.
+
+You should be aware though that the URL will be build in a specific way. If your branch name gets to long, [Let's Encrypt](https://letsencrypt.org/) won't be able to generate a certificate.
```

Comment:
```suggestion
You should be aware though that the URL will be built in a specific way. If your branch name gets too long, [Let's Encrypt](https://letsencrypt.org/) won't be able to generate a certificate.
```

Reply:
Also: Is it known what is "too long"? Could we state the maximum length here?

Reply:
Kind of, it's a combination of multiple thinks. I'd rather link to the documentation instead of listing those points again

https://docs.platform.sh/configuration/routes/https.html#lets-encrypt-limits-errors-and-branch-names

Reply:
Alright I see, I think it's sufficient to link the docs 👍